lib/helper/WebDriverIO.js

```diff
@@ -1422,6 +1639,29 @@ class WebDriverIO extends Helper {
   }
 
   /**
+   * Waits for a specified number of elements on the page
+   */
+  waitNumberOfVisibleElements(locator, num, sec) {
```

Comment:
I think the same can be achieved with `waitForVisible`, you just pass XPath with the number of elements....